.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+:memo: :lipstick: :new: :bomb: :bug: :art: Replace this line with relevant emojis
+
+## Issues or PRs
```

Comment:
`##` is pretty hefty. May want to reduce it a bit. Keep in mind, this is what every issue/pr will look like, too much vericality may be a little annoying.

## This is Two.

### Three

#### Four